mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss

```diff
@@ -1,18 +1,70 @@
 @import "src/styles/variables";
 
+$maxWidth: 960px;
```

Comment:
Why 960?

Reply:
magic number) . changed to 1200px, we can use like our maket in figma 1370px?